main/lsp/notifications/did_change_configuration.cc

```diff
@@ -0,0 +1,30 @@
+#include "main/lsp/notifications/did_change_configuration.h"
+#include "main/lsp/LSPIndexer.h"
+#include "main/lsp/LSPPreprocessor.h"
+
+namespace sorbet::realmain::lsp {
+DidChangeConfigurationTask::DidChangeConfigurationTask(const LSPConfiguration &config,
+                                                       std::unique_ptr<DidChangeConfigurationParams> params)
+    : LSPTask(config, LSPMethod::WorkspaceDidChangeConfiguration), params(move(params)), openFilePaths(), epoch(1) {}
+
+LSPTask::Phase DidChangeConfigurationTask::finalPhase() const {
+    // We want this to run all the way so that the changes to
+    // Global State get propagated through.
+    return LSPTask::Phase::RUN;
+}
+
+void DidChangeConfigurationTask::preprocess(LSPPreprocessor &preprocessor) {
+    auto tmp = preprocessor.openFilePaths();
+    openFilePaths = std::move(tmp);
```

Comment:
What's the point of `tmp` here? can this not be

```suggestion
    openFilePaths = preprocessor.openFilePaths();
```

?

Reply:
Honestly, I don't know. The compiler gets mad about using an implicitly deleted copy constructor if I don't assign to a temporary first. No clue what's up with that.